### PR TITLE
add rails root public dir to jasmine server

### PR DESCRIPTION
**Pull request by diclophis**
@juanca / @jrolfs / @ahuth  / @cantino this will stop alot of the console.log spam when running jasmine

### CODE REVIEW

File: lib/jasmine/server.rb [hunk 1]
```diff
@@ -116,6 +116,7 @@ def self.app(config)
 
       map('/') do
         run Rack::Cascade.new([
+          Rack::URLMap.new('/' => Rack::File.new(Rails.root.join("public"))),
```
Comment (cantino) [inline lib/jasmine/server.rb:119]: `/` can route both to public and to the jasmine dir?

**Comment by diclophis** [comment]
Its a `Rack::Cascade.new`

https://github.com/rack/rack/blob/master/lib/rack/cascade.rb

  # Rack::Cascade tries a request on several apps, and returns the
  # first response that is not 404 or 405 (or in a list of configurable
  # status codes).

**Comment by cantino** [comment]
gotcha

**Comment by ahuth** [comment]
Cool. Not sure of the ramifications of this, but I'm all for removing the Jasmine console spam.

**Comment by diclophis** [comment]
If anything the problems could be something to the effect of having two files named the same, one in our public dir, and one from the jasmine gem dir, and it loads ours instead of loading the jasmine one.

I am going to get a couple more test builds with this added and see if it still timesout or not

**Pull request merged**